src/khoj/routers/email.py

```diff
@@ -47,3 +46,38 @@ async def send_welcome_email(name, email):
             "html": html_content,
         }
     )
+
+
+from pydantic import BaseModel
+
+
+class TextData(BaseModel):
+    uquery: str
+    kquery: str
+    sentiment: str
+
+
+async def feedback(uquery, kquery, sentiment):
+    # console debug messages
```

Comment:
nit: you can remove this

Reply:
Are you referring to the console debug messages?

Reply:
Just the comment!